fhir-search/src/test/java/com/ibm/fhir/search/parameters/ParametersUtilTest.java

```diff
@@ -39,14 +31,17 @@
 public class ParametersUtilTest extends BaseSearchTest {
     public static final boolean DEBUG = false;
 
+    CompartmentUtil compartmentHelper = new CompartmentUtil();
+    ParametersUtil parametersHelper = new ParametersUtil(compartmentHelper);
+
     @Test
     public void testGetAllSearchParameters() throws IOException {
         // Tests JSON
         Set<SearchParameter> params = ParametersUtil.getAllSearchParameters();
         assertNotNull(params);
         // Intentionally the data is captured in the bytearray output stream.
         try (ByteArrayOutputStream outBA = new ByteArrayOutputStream(); PrintStream out = new PrintStream(outBA, true);) {
```

Comment:
should specify the charset encoding here, otherwise this could be affected by the system default charset encoding, which could cause the test to fail.

Reply:
done